advanced_filters/forms.py

```diff
@@ -115,6 +113,11 @@ def _parse_query_dict(query_data, model):
             return query_data
 
         parts = query_data['field'].split('__')
+        boolean_lookups = ['isnull', 'istrue', 'isfalse']
+        for boolean_lookup in boolean_lookups:
+            if query_data['field'].endswith(f'__{boolean_lookup}'):
+                query_data['operator'] = boolean_lookup
```

Comment:
This seems identical to looking up the operator in `AdvancedFilterQueryForm.OPERATORS` (as on line 124 below). Could you explain what this does differently, and what scenario this will address?

Reply:
Correct, should be the same. I don't remember the purpose of these lines of mine but I tested it again and it works exactly as your code so we can delete this part. Thanks for double check.